src/containerapp/azext_containerapp/custom.py

```diff
@@ -3345,6 +3345,18 @@ def list_replicas(cmd, resource_group_name, name, revision=None):
                                             revision_name=revision)
 
 
+def count_replicas(cmd, resource_group_name, name, revision=None):
+    app = ContainerAppClient.show(cmd, resource_group_name, name)
+    if not revision:
+        revision = app["properties"]["latestRevisionName"]
```

Comment:
please use `safe_get()` to avoid `KeyError`.
And handle error for `containerapp` which doesn't have property `latestRevisionName`.

Reply:
You're right, added it in.